src/common/transformations/include/transformations/op_conversions/reduce_modification_for_false_keepdims.hpp

```diff
@@ -0,0 +1,143 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
As this is gpu-specific, I think correct place is not `src/common/transformations` but `src/plugins/intel_gpu/plugins/transformations`

Reply:
Moved